modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -127,13 +127,24 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 [[bug-fixes]]
 == Bug fixes
 
-{productname} <x.y[.z]> also includes the following bug fix<es>:
+{productname} 7.0 also includes the following bug fix<es>:
 
 === <TINY-vwxyz 1 changelog entry>
 //#TINY-vwxyz1
 
 // CCFR here.
 
+=== `TableOperations.opPasteColsAfter` & `TableOperations.opPasteRowsAfter` now place the cursor in the newly pasted row/column to be consistent with the other pasting operations.
+// #TINY-6460
+
+Previously, the cursor position in `opPasteRowsAfter` and `opPasteColsAfter` was calculated incorrectly after pasting a new rol or column.
+
+As a result, after pasting, the cursor remained in the same position instead of moving to the newly pasted row or column.
+
+{productname} 7.0 addresses this, fixing the cursor calculation in `opPasteRowsAfter` and `opPasteColsAfter` to use the 'index' variable.
+
+Now, when using "Paste row/column after," the cursor is correctly positioned in the newly pasted row or column.
```

Comment:
```suggestion
Previously, the cursor position in `opPasteRowsAfter` and `opPasteColsAfter` was calculated incorrectly after pasting a new row or column.

As a consequence, after pasting, the cursor remained in the same position instead of moving to the newly pasted row or column.

{productname} 7.0 addresses this, fixing the cursor calculation in `opPasteRowsAfter` and `opPasteColsAfter` to use the `index` variable.

Now, when using `opPasteRowsAfter` and `opPasteColsAfter`, the cursor is correctly positioned in the newly pasted row or column as expected.
```

Reply:
- [ ] @danoaky-tiny Can you confirm we are ok to refer to `opPasteRowsAfter` and `opPasteColsAfter` and `TableOperations.opPasteColsAfter` & `TableOperations.opPasteRowsAfter` in public facing documentation before this ticket is `merged` please.

Reply:
@kemister85 It's not public facing no